src/pages/code-for-good.js

```diff
@@ -0,0 +1,304 @@
+import React from 'react';
+import Layout from '../components/dev-hub/layout';
+import { Helmet } from 'react-helmet';
+import { useSiteMetadata } from '../hooks/use-site-metadata';
+import {
+    colorMap,
+    fontSize,
+    screenSize,
+    size,
+    lineHeight,
+} from '../components/dev-hub/theme';
+import { H3, ArticleH2, P } from '../components/dev-hub/text';
+import styled from '@emotion/styled';
+import Button from '../components/dev-hub/button';
+import HeroBanner from '../components/dev-hub/hero-banner';
+import BackgroundWaves from '../images/code-for-good/background-waves.png';
+import WildAidLogo from '../images/code-for-good/wild-aid-logo.png';
+import WildAidDashboard from '../images/code-for-good/wild-aid-dashboard.png';
+import VesselRecord from '../images/code-for-good/vessel-record.png';
+
+const CAPTION_MAX_WIDTH = '345px';
+const BODY_TEXT_MAX_WIDTH = '320px';
+const HERO_TEXT_WIDTH = '425px';
+const DASHBOARD_IMAGE_WIDTH = 780;
+const VESSEL_IMAGE_WIDTH = 345;
+const LOGO_IMAGE_WIDTH = 350;
+const LOGO_IMAGE_HEIGHT = 450;
+
+const Header = styled('header')`
+    display: flex;
+    justify-content: space-between;
+    padding-top: ${size.large};
+    @media ${screenSize.upToLarge} {
+        display: block;
+        margin-bottom: ${size.large};
+    }
+`;
+const HeaderText = styled('div')`
+    margin-bottom: -${size.xlarge};
+    max-width: ${HERO_TEXT_WIDTH};
+`;
+
+const HeaderLink = styled('a')`
+    color: ${colorMap.darkGreen};
+    margin-top: ${size.large};
+    margin-left: ${size.mediumLarge};
+    text-decoration: none;
+    @media ${screenSize.upToLarge} {
+        display: block;
+        margin-left: auto;
+    }
+`;
+
+const Title = styled(ArticleH2)`
+    font-size: ${fontSize.small};
+    font-family: 'Fira Mono';
+`;
+
+const StyledHeroBanner = styled(HeroBanner)`
+    margin: 0 ${size.medium};
+`;
+
+const StyledP = styled(P)`
+    font-family: akzidenz;
+    font-size: ${fontSize.default};
+    line-height: ${lineHeight.default};
+    margin-top: ${size.articleContent};
+    margin-bottom: ${size.xlarge};
+`;
+
+const BodyText = styled(P)`
+    font-family: akzidenz;
+    font-size: ${fontSize.default};
+    line-height: ${lineHeight.default};
+    margin-top: ${fontSize.default};
+`;
+
+const StyledButton = styled(Button)`
+    margin-top: ${size.large};
+    margin-bottom: ${size.xlarge};
+    @media ${screenSize.upToLarge} {
+        margin-bottom: ${size.small};
+    }
```

Comment:
This should be on the Header and not on the Button, since the Header is controlling the layout for its children (that will help some odd spacing issues with the link on smaller devices)